### DISCUSSION TITLE
Support: restore the system default window size and position (original)

### DISCUSSION THREAD

**Discussion by snowstormfloratracks** [rxhanson/Rectangle#484] [Ideas]
Hello,

I am using this current Rectangle version https://github.com/rxhanson/Rectangle/releases/tag/v0.46.

The function "**Restore**" that is displayed there in Rectangle restores only the last saved window size and position.

![Bildschirmfoto 2021-06-10 um 17 41 05](https://user-images.githubusercontent.com/85694775/121555183-132f7a00-ca13-11eb-92bf-21449faeebdd.png)

But I want to restore the original default window size and position for my apps as if I start the apps for the first time.

Is it possible to support this? 

These are the apps I use almost every day: 
- Firefox
- IINA

Apple Apps
- Safari
- Finder (when local folders are opened on the Desktop)
- Calendar
- Maps
- Chess
- DVD-Player

**Comment by rxhanson** [comment]
Thanks for asking. This isn't something that I am looking to add into Rectangle (sorry!).

If you would like to fork Rectangle and add this functionality, I don't mind giving some pointers as to where to start.
